netlify.toml

```diff
@@ -35,17 +35,17 @@ status = 301
 
 [[redirects]]
 from = "/contracts-cairo/3.0.0/*"
-to = "/contracts-cairo/:splat"
+to = "/contracts-cairo/alpha/:splat"
```

Comment:
Why do we even have this redirect if we haven't released 3.0.0 yet?

Reply:
That might have been a typo, we can remove it